### PR TITLE
[Pyrrans] AC-3208: updated symfony/console to the latest version

### PR DESCRIPTION
**Pull request by anzin**
# Description (*)

This pull request (PR) provides updated symfony/console to the latest version 5.4.9

## Fixed Issues (if relevant)

1. Fixes [AC-3208](https://jira.corp.magento.com/browse/AC-3208)

## Contribution checklist (*)

- [ ] Pull request has a meaningful description of its purpose
- [ ] All commits are accompanied by meaningful commit messages
- [ ] All new or changed code is covered with unit/integration tests (if applicable)
- [ ] All automated tests passed successfully (all builds are green)

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -15,7 +15,7 @@
         "nikic/php-parser": "~4.12.0",
         "phpstan/phpdoc-parser": "^0.5.5",
         "sabre/xml": "~2.2.3",
-        "symfony/console": "~4.1.0||~4.4.0",
+        "symfony/console": "~4.1.0||~4.4.0||~5.4.0",
```
Comment (xmav) [inline composer.json]: @anzin Can we drop 4.1.x ?

**Comment by anzin** [comment]
@magento import pr to magento-commerce/magento-semver

**Comment by m2-github-services** [comment]
@anzin the Pull Request is successfully imported.

File: composer.json (hunk 1 above)
Comment (anzin) [reply composer.json]: Hello @xmav, I dropped 4.1.x thank you for review.

**Pull request closed without merging**